app/inithttp.go

```diff
@@ -251,7 +251,7 @@ func (app *App) initHTTP(ctx context.Context) error {
 	mux.HandleFunc("/health", app.healthCheck)
 	mux.HandleFunc("/health/engine", app.engineStatus)
 
-	webH, err := web.NewHandler(app.cfg.UIURL, app.cfg.HTTPPrefix)
+	webH, err := web.NewHandler(app.cfg.UIURL, app.cfg.HTTPPrefix, app.ConfigStore.Config().General.ApplicationName)
```

Comment:
This needs to come from context on-demand as inithttp is only run once.

We'll need to alter how we handle index.html and the graphql playground so we pull this from the request context -- otherwise, config changes won't take effect until instances are manually restarted.